alpm-utils/src/depends.rs

```diff
@@ -1,11 +1,7 @@
 use alpm::{AsDep, DepModVer, Ver};
 
 /// Checks if a dependency is satisfied by a package (name + version).
-pub fn satisfies_dep<S: AsRef<str>, V: AsRef<Ver>>(
-    dep: impl AsDep,
-    name: S,
-    version: V,
-) -> bool {
+pub fn satisfies_dep<S: AsRef<str>, V: AsRef<Ver>>(dep: impl AsDep, name: S, version: V) -> bool {
```

Comment:
Have you got some custom cargo fmt config?

Reply:
I don't, but the last changes to these lines occurred in 2019, so there's a chance that `rustfmt` changed since then.